### DISCUSSION TITLE
Disabled JS Users

### DISCUSSION THREAD

**Discussion by kilometerlime** [gatsbyjs/gatsby#34907] [Help]
Hey all, we're currently trying to get a project working for users with JS disabled. For now we're just displaying a simple disclaimer with a `noscript` tag when JS is disabled. However, we'd like the actual content of the website to still be accessible to these users. Currently with JS disabled the site just displays a completely blank page with no errors in the console nor any errors in the node output from npm run serve.

I've had a look at some other gatsby sites featured on the gatsby showcase page, some continue to work with JS disabled while others do not which hopefully means its just an implementation issue on my end. Any information or insight into how we can achieve this would be greatly appreciated, please reach out if you need more info, thank you!

**Comment by tyhopp** [comment]
Hi @milesbigbluedoor, I tried this out with the [minimal starter](https://github.com/gatsbyjs/gatsby/tree/master/starters/gatsby-starter-minimal) and was able to see the site content when running `yarn build` and `yarn serve`.

If you're using the [default static generation rendering mode](https://www.gatsbyjs.com/docs/conceptual/rendering-options/#static-site-generation-ssg) the HTML is viewable with JS disabled. Hard to say more without looking at your project but hopefully that's helpful!
